README.md

```diff
@@ -38,6 +38,10 @@ Ceph contains many git submodules that need to be checked out with
 
 ## Build Prerequisites
 
+A debug build of Ceph with the source repo takes around 40 gigabytes. If you want to build Ceph in a virtual machine (VM) please make sure total disk space on the VM is at least 60 gigabytes.
```

Comment:
A debug build for Ceph may take around 40 gigabytes
```suggestion
A debug build of Ceph may take around 40 gigabytes. If you want to build Ceph in a virtual machine (VM) please make sure total disk space on the VM is at least 60 gigabytes.
```

Reply:
hey @jbwyatt4 looks good! 
can you squash all commits into 1 keeping commit message as https://github.com/ceph/ceph/pull/33975#issue-388455780 just changing `added to "Build Prerequisites"` to `added space requirement to "Build Prerequisites" `

Reply:
Will do @ideepika.

Reply:
> hey @jbwyatt4 looks good!
> can you squash all commits into 1 keeping commit message as [#33975 (comment)](https://github.com/ceph/ceph/pull/33975#issue-388455780) just 

> changing `added to "Build Prerequisites"` to `added space requirement to "Build Prerequisites" `
looks good, can you update commit title?
you can do: 
```
git commit --amend
```
to change commit message

Reply:
Understood. Amended and force pushed.